internal/web/middleware/security.go

```diff
@@ -217,6 +217,11 @@ func checkAllAuthentication(ctx context.Context, method string, urlPath string,
 		}
 	}
 
+	// countdown endpoint is allowed through (but AFTER auth processing)
+	if method == http.MethodGet && urlPath == "/api/rest/v1/countdown" {
```

Comment:
hardcoded path is not optimal, for instance when there is a version change. Maybe add a list of allowed endpoints to the config to make it at least configurable

Reply:
good point, especially when it's in the middle of a function.

I don't really like giving the deployment the option to mess up security, but I've at least moved it to an easy to find location.